backend/scripts/mirror_rds_data.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+# Mirror RDS (postgres db) data from a source deployment environment (usually production)
+# to a specified destination deployment environment (dev or staging). Steps:
+# 1. Dump the database using pg_dump from the src env
+# 2. Loads the pg_dump data file into a PostgreSQL db in the dest env
+# 3. Updates table columns that contain deployment environment-specific URLs to point at the dest environment
+#
+# THIS IS DESTRUCTIVE for the dest env! The src env database will never be modified, but dest environment database will
+# be replaced. The destination database will be dumped to a local file, for peace of mind.
+
+set -e
+
+SCRIPTS_DIR=`dirname $0`
+. $SCRIPTS_DIR/set_src_dest_envs.sh
+
+echo Mirroring RDS data from $SRC_ENV to $DEST_ENV
+
+kill_ssh_tunnel()
+{
+  pkill -f "bastion\..*\.single-cell\.czi\.technology" || true
+}
+
+DB_DUMP_FILE=`mktemp`
+
+export DEPLOYMENT_STAGE=$SRC_ENV
+if [[ $SRC_ENV == 'staging' ]]; then
+   export AWS_PROFILE=single-cell-dev
+else
+   export AWS_PROFILE=single-cell-${SRC_ENV}
+fi
+# TODO: move kill ssh tunnel to Makefile db/tunnel command
+kill_ssh_tunnel
```

Comment:
What do you think of moving the kill_ssh_tunnel command into its own make command (instead of the db/tunnel command)? Then you could call it here and we could also update the backend readme to recommend calling it after  completing any work with the db as a safety measure